tests/run.sh

```diff
@@ -29,6 +29,12 @@ run_tox() {
     tox -r "$@"
 }
 
+run_trogdor() {
+    start_cluster
+    echo "Executing Trogdor"
+    python ${TEST_SOURCE}/trogdor/trogdor_runner.py --spec ${TEST_SOURCE}/trogdor/example-produce-spec.json
```

Comment:
this should not point to the example, but a proper file. The user/dev should copy and modify the example file into place. Alternatively we call this file `produce-spec.json`, without the example part, to allow it to run out of the box.